packages/workbox-core/src/_private.ts

```diff
@@ -20,12 +21,14 @@ import {getFriendlyURL} from './_private/getFriendlyURL.js';
 import {logger} from './_private/logger.js';
 import {resultingClientExists} from './_private/resultingClientExists.js';
 import {timeout} from './_private/timeout.js';
+import {waitUntil} from './_private/waitUntil.js';
 import {WorkboxError} from './_private/WorkboxError.js';
 
 import './_version.js';
 
 export {
   assert,
+  cacheMatchIgnoreParams,
```

Comment:
Is `cacheMatchIgnoreParams()` calling anywhere internally, or is it just meant as a new, externally-visible helper method? I don't see it in the RFC.

As mentioned later, it's awkward that it takes in an array of strings and not RegExps, as RegExps are used elsewhere for a similar purpose.